zenodo/modules/spam/templates/zenodo_spam/delete.html

```diff
@@ -104,7 +104,7 @@
         <div class="panel-body">
           User has <strong>{{ rec_count }}</strong> records indexed in elasticsearch.
 
-          <a href="{{ url_for('invenio_search_ui.search', q='owners: ' + (user.id|string)) }}"><strong>Full record search result</strong></a>
+          <a href="{{ url_for('invenio_search_ui.search', q=('owners:' + (user.id|string))) }}"><strong>Full record search result</strong></a>
```

Comment:
curious - what was the bug here?

Reply:
I think the space didn't allow the user ID to be parsable (ie as `owners:12345`, but rather just a generic argument (ie. `owners:` and `12345`), so results weren't consistent.